File: VisitBosnia/VisitBosnia.Model/Requests/PostReplyInsertRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.Requests
{
    public class PostReplyInsertRequest
    {
        public int AppUserId { get; set; }
        public int PostId { get; set; }
        public DateTime CreatedTime { get; set; }
        public string Content { get; set; } = null!;

    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/postReply/post_reply_insert_request.dart
import 'package:visit_bosnia_mobile/model/appUser/app_user.dart';

class PostReplyInsertRequest {
  int? appUserId;
  int? postId;
  String? createdTime;
  String? content;

  PostReplyInsertRequest({
    this.appUserId,
    this.postId,
    this.createdTime,
    this.content,
  });

  PostReplyInsertRequest.fromJson(Map<String, dynamic> json) {
    appUserId = json['appUserId'];
    postId = json['postId'];
    createdTime = json['createdTime'];
    content = json['content'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['appUserId'] = this.appUserId;
    data['postId'] = this.postId;
    data['createdTime'] = this.createdTime;
    data['content'] = this.content;
    return data;
  }
}
